packages/types/src/proving-keys.ts

```diff
@@ -5,5 +5,5 @@ export const provingKeys = [
   { keyType: 'swap_claim', file: 'swapclaim_pk.bin' },
   { keyType: 'nullifier_derivation', file: 'nullifier_derivation_pk.bin' },
   { keyType: 'delegator_vote', file: 'delegator_vote_pk.bin' },
-  { keyType: 'undelegate_claim', file: 'undelegateclaim_pk.bin' },
+  { keyType: 'undelegate_claim', file: 'convert_pk.bin' },
```

Comment:
See: https://github.com/penumbra-zone/web/pull/354

Reply:
i haven't been able to identify any errors, but: is it correct that the filename changes, but keyType remains the same?

Reply:
keyType should be named `convert` according to the changes in https://github.com/penumbra-zone/penumbra/pull/3579/files